contributing/documentation/content_guidelines.rst

```diff
@@ -160,6 +160,42 @@ A few tips to improve your screenshots:
    Resizing the window's width is the most important step to do as Odoo's responsive design
    automatically resizes all fields to match the window's width.
 
+.. _contributing/media-files:
+
+Media files
+-----------
+
+A **media filename** is written in **lower-case letters** and is composed of two parts:
+
+#. An **identifying name** relevant to the media's content. E.g., ``awesome-filename``.
+
+   .. important::
+      Use a **hyphen** ``-`` to separate the words in the media filename.
+
+#. The **file extension**, which is ``.png`` for most images.
```

Comment:
This is not wrong but don't you think that it may add confusion for nothing?
If someone is asked to insert "duck" in every filename I have faith that I'll never see a `is-this-correct.duckpng` file.

Reply:
😅